src/test/scala/ru/smslv/akka/dns/AsyncDnsResolverSpec.scala

```diff
@@ -76,6 +78,14 @@ class AsyncDnsResolverSpec extends AkkaSpec(
       ))
       answer.ipv6.toSet should be(empty)
     }
+    "resolve SRV record" in {
+      val answer = Await.result(IO(Dns) ? Dns.Resolve("_http._tcp.mxtoolbox.com"), duration).asInstanceOf[SrvResolved]
```

Comment:
I've created an SRV record inside a smslv.ru domain, could you please use it instead for consistency?
I've exported relevant smslv.ru zone records into `/test.zone`

```
host -t SRV _http._tcp.smslv.ru
_http._tcp.smslv.ru has SRV record 10 15 80 a-single.test.smslv.ru.
```